apostrophecms/apostrophe
# Pro 3515 edit permission

## Summary

Add `editPermission` field

## What are the specific steps to test this change?

1. `npm test`
2. Create a piece type schema with some fields with `editPermission` as `{ action: 'edit', type: '@apostrophecms/user' }`
3. Open the manager modal for the piece and create a new piece as an admin
4. Connect as an editor or a contributor and edit the piece with the network panel open
5. Edit the `PUT` request and add the field for which the current user does not have write permission
6. Check that the request contains the updated field and the response the initial value as done in the unit tests.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
